Automattic/wp-calypso
# Account Closure: Provide Confirmation On-Screen

#### Changes proposed in this Pull Request

Currently, when an account is closed, there's no confirmation that the account has been closed, and things can break if you start clicking away during the process. This PR changes that. It probably isn't exactly ideal since it creates a new section, but I found it the best way to make sure there's no sidebar or masterbar.

#### Testing instructions

Visit `me/account/closed` directly and verify you're taken to `me/account/close`. Now try to close your account following the on-screen steps, and verify that the result is the same as the GIF below.

**Before:**

![ezgif-4-f3cb09815a4e](https://user-images.githubusercontent.com/43215253/69093511-7bd88f80-0a46-11ea-8030-b7458096f871.gif)

**After:**

![ezgif-4-38189b5c2a99](https://user-images.githubusercontent.com/43215253/69093584-a32f5c80-0a46-11ea-875b-e7c158fc2dbd.gif)

cc @bluefuton - last account closure PR I think :)

Fixes #35977
